application/helpers/update/updates/Update_641.php

```diff
@@ -1305,7 +1305,7 @@ public function up()
                 foreach ($ef['entities'] as $entity) {
                     $save = [
                         $this->handleInsertans($entity, $ef['fields'], $sid),
-                        $this->fixText($entity, $ef['fields'], $names),
+                        isset($names) && $this->fixText($entity, $ef['fields'], $names),
```

Comment:
This was a typo indeed, but you do not need to check whether $names has been set. Instead, try using $newFields